src/api/app/assets/javascripts/webui/request.js

```diff
@@ -110,7 +110,7 @@ function requestAddAutocomplete(autocompleteElement) { // jshint ignore:line
 
 $(document).ready(function(){
   var element = $('.bs-request-actions li:first-child a:first-child');
-  if (element['length'] !== 0){
+  if (element.length !== 0){
```

Comment:
This change belongs to the previous commit ` Improve performance of Request#show with multiple actions`.

Reply:
Done. It's moved to the correct commit.